src/Pages/result/Components/Data.tsx

```diff
@@ -0,0 +1,259 @@
+  },
+  reButton: {
+    height: "45px",
+    width: "365px",
+    backgroundColor: "#3A3A3D",
+    color: "#ffffff",
+    textTransform: "none",
+    borderRadius: "20px",
+    fontSize: "20px",
+    fontWeight: 700,
+    marginBottom: "16px",
+    "&:hover": { backgroundColor: "#3A3A3D", color: "#ffffff" },
+  },
+}));
+
+const TextBox = withStyles(() => ({
+  root: {
+    padding: "16px",
+  },
+}))(Box);
+
+const Text = withStyles(() => ({
+  root: {
+    marginBottom: "16px",
+  },
+}))(Box);
+
+const BorderBox = withStyles(() => ({
+  root: {
+    border: "2px solid #d3d3d3",
+    borderRadius: 10,
+    padding: "16px",
+    marginBottom: "16px",
+  },
+}))(Box);
+
+const TextPinkBox = withStyles(() => ({
+  root: {
+    fontWeight: 600,
+    color: "#CF658D",
+    marginBottom: "8px",
+    alignContent: "center",
+    justifyContent: "center",
+    display: "flex",
+  },
+}))(Typography);
+
+const TextBlackBox = withStyles(() => ({
+  root: {
+    fontWeight: 600,
+    alignContent: "center",
+    justifyContent: "center",
+    display: "flex",
+    color: "#3A3A3D",
+  },
+}))(Typography);
+
+const TextBlackTitle = withStyles(() => ({
+  root: {
+    marginBottom: "8px",
+    color: "#3A3A3D",
+  },
+}))(Typography);
+
+const TextBlack = withStyles(() => ({
+  root: {
+    fontWeight: 400,
+    fontSize: "18px",
+    color: "#3A3A3D",
+    marginLeft: "8px",
+  },
+}))(Typography);
+
+interface DataProps {
+  name: string;
+  patientID: string;
+  gender: string;
+  age: number;
+  arrivalDate: string;
+  totalTestsDone: number;
+  timeCourse: string;
+  onsetDate: string;
+  duration: string;
+  symptoms: string[];
+  underlyingDiseases: string[];
+  systolicBP: number;
+  diastolicBP: number;
+  heartRate: number;
+  heartRateText: string;
+  EKG12Leads: string;
+  NIHSS: number;
+}
+
+interface Props {
+  data: DataProps;
+}
+
+const Data = (props: Props) => {
+  const classes = useStyles();
+  const { data } = props;
+  const {
+    name,
+    patientID,
+    gender,
+    age,
+    arrivalDate,
+    totalTestsDone,
+    timeCourse,
+    onsetDate,
+    duration,
+    symptoms,
+    underlyingDiseases,
+    systolicBP,
+    diastolicBP,
+    heartRate,
+    heartRateText,
+    EKG12Leads,
+    NIHSS,
+  } = data;
+
+  return (
+    <Paper elevation={0} className={classes.root}>
+      <Typography className={classes.name} variant="h5">
+        {name}
+      </Typography>
+      <Box className={classes.box}>
+        <Toolbar>
+          <TextBox>
+            <TextPinkBox variant="subtitle1">Patient ID</TextPinkBox>
+            <TextBlackBox variant="h6">{patientID}</TextBlackBox>
+          </TextBox>
+          <TextBox>
+            <TextPinkBox variant="subtitle1">Gender</TextPinkBox>
+            <TextBlackBox variant="h6">{gender}</TextBlackBox>
+          </TextBox>
+          <TextBox>
+            <TextPinkBox variant="subtitle1">Age</TextPinkBox>
+            <TextBlackBox variant="h6">{age}years</TextBlackBox>
```

Comment:
Add space between {age} years.

Reply:
It's 2 pictures next to each other.

<img width="321" alt="ภาพถ่ายหน้าจอ 2564-08-15 เวลา 11 39 59" src="https://user-images.githubusercontent.com/59549553/129467355-ce24cb0a-f878-4fb9-ab95-b46ce8881321.png">
//////////////////////////////////////////////////////////////////////////////
<img width="323" alt="ภาพถ่ายหน้าจอ 2564-08-15 เวลา 11 39 06" src="https://user-images.githubusercontent.com/59549553/129467340-4b40f945-39d8-4354-bb5d-123974ccd357.png">